tests/support/run-tests.sh

```diff
@@ -8,52 +8,65 @@ script_directory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 test_directory="${script_directory}/.."
 
 
+if env | grep -q '^HELM_VERSION=' 2>/dev/null; then
+	HELM_VERSION="$( env | grep '^HELM_VERSION=' | sed 's/^HELM_VERSION=//g' )"
+else
+	HELM_VERSION="2.12.3"
+fi
+
+if env | grep -q '^ANSIBLE_ARGS=' 2>/dev/null; then
+	ANSIBLE_ARGS="--diff $( env | grep '^ANSIBLE_ARGS=' | sed 's/^ANSIBLE_ARGS=//g' )"
+else
+	ANSIBLE_ARGS="--diff"
+fi
+
+
+echo "${ANSIBLE_ARGS}"
+echo
+echo "----------------------------------------------------------------------------------------------------"
+echo "- [NEED TO FAIL] Helm is not installed"
+echo "----------------------------------------------------------------------------------------------------"
 # --- Test if role fails when Helm command does NOT exist ---
-set +e
 # Test if helm was installed successfully
-ansible-playbook "${test_directory}/test-defaults.yml"
-
-exit_code=$?
-if [ "${exit_code}" = '0' ]; then
+if ansible-playbook ${ANSIBLE_ARGS} ${test_directory}/test-defaults.yml; then
   echo "Role should fail since helm command does NOT exist"
   exit 1
 fi
-set -e
-# ---
 
 
+echo
+echo "----------------------------------------------------------------------------------------------------"
+echo "- [NEED TO FAIL] Helm has wrong version"
+echo "----------------------------------------------------------------------------------------------------"
 # --- Test if role fails when specific Helm version does NOT exist ---
 # Mock Helm
 echo -e "#!/usr/bin/env bash" > /usr/local/bin/helm
-echo -e "echo '2.12.3-wrong-version'" >> /usr/local/bin/helm
+echo -e "echo '1-wrong-version'" >> /usr/local/bin/helm
```

Comment:
Why `1-wrong-version`?

Reply:
I need to ensure that this fails for both:

* strict version check
* loose version check

With the previous one `2.12.3-wrong-version`, it would have checked if `2.12` is set and then had passed.

Reply:
:+1: